scripts/import-db-neo4j

```diff
@@ -258,6 +275,19 @@ MATCH (n1:SDG)
 WHERE n1.Name = i
 MERGE (n0)-[:ADDRESSES]->(n1);
 
+// Paper -> CONCEPTUALIZE -> Concept
```

Comment:
Depending on the constraints for the syntax here and how the file is consumed, consider adding a quick summary which is human-readable about what this and the block below will perform.